feat(gatsby-plugin-image): Add helper function to grab src (#28463)
gatsbyjs/gatsby: packages/gatsby-plugin-image/README.md

`
```

> If you need the image `src` directly you can use the `getSrc` helper function, which is equivalent to `data.file.childImageSharp.gatsbyImageData.images.fallback.src`.

Author: ```suggestion
If you need the image `src` directly you can import the `getSrc` helper function from `gatsby-plugin-image`. That function is equivalent to `data.file.childImageSharp.gatsbyImageData.images.fallback.src`.
```
